app/src/main/java/com/hillnerds/soundsparrow/ChannelSoundGeneration.java

```diff
@@ -26,13 +26,22 @@
             new Instrument("flute", 74, 73, 90)
     };
 
+     // (1100 0000) the first 4 bits of a starting sequence for an instrument change
+    final static int INSTRUMENT_CHANGE_CODE = 192;
+
+    //the amount of notes generated for each channel
+    final static int CHANNEL_NOTE_AMOUNT = 16;
+
+    // (1001 0000) the first 4 bits of a note on sequence
+    final static int NOTE_ON_START_CODE = 144;
+
     /**
      * This function chooses a rondom instrument form a hardcoded instrumentList.
      * @param random - a random generator Object
      * @return - return an Instrument object from instrumentList
      */
     public static Instrument generateRandomInstrument(Random random){
-        int instrumentNumber = random.nextInt((6 - 0) + 1) + 0;
+        int instrumentNumber = random.nextInt((instrumentList.length - 0) + 1) + 0;
```

Comment:
You don't need length - 0 +1
nextInt will generate an int in range (0, length], meaning you will get a valid index with just that. I think this will cause an occasional OutOfBoundsException
`int instrumentNumber = random.nextInt(instrumentList.length);`